tests/functional/disaster-recovery/regional-dr/test_uninstall_dr.py

```diff
@@ -0,0 +1,157 @@
+
+        primary_cluster_name = dr_helpers.get_current_primary_cluster_name(
+            workloads[0].workload_namespace
+        )
+        config.switch_to_cluster_by_name(primary_cluster_name)
+
+        secondary_cluster_name = dr_helpers.get_current_secondary_cluster_name(
+            workloads[0].workload_namespace
+        )
+
+        # Get dr policy name associated with primary and secondary cluster
+        dr_policy_name = dr_helpers.get_drpolicy(
+            primary_cluster_name, secondary_cluster_name
+        )
+
+        # Disable RDR on all workloads
+        logger.info("Disabling RDR on all the workloads...")
+        dr_helpers.disable_dr_rdr()
+
+        # Get mirror peer assosiated with dr policy
+        mirrorpeer_name = dr_helpers.get_mirrorpeer(dr_policy_name)
+
+        # Perform partial DR Uninstall
+        logger.info(
+            f"Deleting the dr policy {dr_policy_name} "
+            f"and mirror peer {mirrorpeer_name} from ACM"
+        )
+        dr_helpers.partial_rdr_uninstall(dr_policy_name, mirrorpeer_name)
+
+        # Validate mirroring status in radospoolnamespace
+        # TODO: skipping this for now due to existing bug 2775
+
+        # Validate mirroring in tools pod
+        logger.info("Validating the mirroring status in tools pod...")
+
+        expected_mirroring_status = [
+            "Mode: disabled",
+            "rbd: mirroring not enabled on the pool",
+        ]
+        dr_helpers.verify_mirroring_status_in_tools_pod(expected_mirroring_status)
+
+        # Check ceph health
+        logger.info("Checking for Ceph Health OK")
+        ceph_health_check()
+
+        # Complete RDR Uninstall on ACM
+        logger.info("Proceeding to complete RDR uninstallation...")
+        dr_helpers.complete_rdr_uninstall()
+
+        # Validate complete uninstall
+        logger.info(
+            f"Verifying the RDR resource deletion on the cluster {primary_cluster_name}"
+        )
+        dr_helpers.verify_resource_rdr_resource_deletion(primary_cluster_name)
+
+        logger.info(
+            f"Verifying the RDR resource deletion on the cluster {secondary_cluster_name}"
+        )
+        dr_helpers.verify_resource_rdr_resource_deletion(secondary_cluster_name)
+
+        # Reinstall RDR
+        # Install MCO on the hub again
+        from ocs_ci.deployment.deployment import MultiClusterDROperatorsDeploy
+
+        dr_conf = dict()
+        mco_obj = MultiClusterDROperatorsDeploy(dr_conf)
+
+        mco_obj.deploy()
+        mco_obj.configure_mirror_peer()
+        mco_obj.deploy_dr_policy()
+        dr_helpers.apply_drpolicy_to_workload(workloads, primary_cluster_name)
+
+        # Validate Resource creation
+        for wl in workloads:
+            dr_helpers.wait_for_all_resources_creation(
+                wl.workload_pvc_count,
+                wl.workload_pod_count,
+                wl.workload_namespace,
+            )
```

Comment:
We can add verification of the mirroring status and lastGroupSyncTime after reinstalling RDR.

Reply:
done